### PR TITLE
docs: bulk add/remove users

### PR DESCRIPTION
**Pull request by jorwoods**

### CODE REVIEW

File: docs/api-ref.md [hunk 1]
```diff
@@ -1244,6 +1294,56 @@ None. The user is removed from the group.
...
+```
+
+Removes multiple users to the specified group.
+
+
+REST API [Remove User from Group](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref.htm#remove_user_to_group)
+
+**Version**
+
+This endpoint is available with REST API version 3.21 and up. Method removeed in
```
Comment (bcantoni) [inline docs/api-ref.md:1310]: I think you meant "Method added in..." here

File: docs/api-ref.md [hunk 2]
```diff
@@ -1244,6 +1294,56 @@ None. The user is removed from the group.
 <br>
 <br>
 
+#### groups.remove_users
+
+```py
+groups.remove_users(group_item, user_id):
+```
+
+Removes multiple users to the specified group.
```
Comment (bcantoni) [inline docs/api-ref.md:1303]: "...from the"

File: docs/api-ref.md (hunk 1 above)
Comment (jorwoods) [reply docs/api-ref.md]: Fixed.

File: docs/api-ref.md [hunk 3]
```diff
@@ -1244,6 +1294,56 @@ None. The user is removed from the group.
...
+
+**Returns**
+
+None
+
+
+**Example**
+
+```py
+# Removing a user to a group
```
[the remark this answers is not in the document]
Comment (jorwoods) [reply docs/api-ref.md]: Fixed.

File: docs/api-ref.md (hunk 2 above)
Comment (jorwoods) [reply docs/api-ref.md]: Fixed.

**Review by jacalata — APPROVED** [review]

**Review by bcantoni — APPROVED** [review]

**Pull request merged**